<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1358</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0,0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; color:#ffaa00;&quot;&gt;KUKA BARTENDER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>1181</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,0, 0);

</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="button_bebida">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>341</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/coktail.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="button_manual">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>341</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/tbebida.jpeg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button_clean">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>30</y>
        <width>341</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/limpeza.jpeg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="cuba_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>201</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/cuba.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="gin_button">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>201</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/gin.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="vodka_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>201</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/bebida.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="caipirinha_button">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>250</y>
        <width>201</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/caipirinha.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffaa00;&quot;&gt;Bebida:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_bebida">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>50</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>110</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffaa00;&quot;&gt;Ingredientes:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Ingredientes">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>160</y>
        <width>171</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_ok">
      <property name="geometry">
       <rect>
        <x>1042</x>
        <y>347</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/ok.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_preparar">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>360</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>301</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffaa00;&quot;&gt;Faz a tua bebida&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_wisckey">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>151</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/wisckey.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button_limpar">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>370</y>
        <width>111</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/lixo.jpeg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button_bebida_ok">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>400</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/ok.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_receita">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>150</y>
        <width>251</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lista de ingredientes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_cola">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>151</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/coca_cola.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button_gin">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>151</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/gin_bottle.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button_tonica">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>300</y>
        <width>151</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/tonic.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button_vodka">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>151</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-image: url(:/bw-absolut-vodka-1.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button_redbul">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>300</y>
        <width>151</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/redbull.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>100</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffaa00;&quot;&gt;Ingredientes Selecionados:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_bebida_a">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>320</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>30</y>
        <width>581</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 182, 182);</string>
      </property>
      <property name="text">
       <string>IMAGEM DA CAMERA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffaa00;&quot;&gt;Copos Identificados&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">

background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>340</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/copo.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_menu">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>10</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Home-512.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1358</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionASD">
   <property name="text">
    <string>ASD</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
